core/src/main/resources/org/apache/spark/ui/static/executorspage-template.html

```diff
@@ -24,7 +24,7 @@ <h4 style="clear: left; display: inline-block;">Summary</h4>
         <th></th>
         <th>RDD Blocks</th>
         <th><span data-toggle="tooltip"
-                  title="Memory used / total available memory for storage of data like RDD partitions cached in memory. ">Storage Memory</span>
+                  title="Memory used / total available memory for storage of data like RDD partitions cached in memory. ">Storage Memory used/total</span>
```

Comment:
Let's post before/after snapshoots here and defer to committers. Personally, I am not sure too because tooltip explains quite clear.

BTW, I think we should capitalise it just to be consistent other pages.

![2017-03-30 6 21 24](https://cloud.githubusercontent.com/assets/6477701/24497335/0f62dabc-1576-11e7-9ec4-ad0f3cd8206c.png)
![2017-03-30 6 21 18](https://cloud.githubusercontent.com/assets/6477701/24497339/15032e36-1576-11e7-99d7-8493eea3ae92.png)
![2017-03-30 6 21 15](https://cloud.githubusercontent.com/assets/6477701/24497341/1810be18-1576-11e7-907f-858f692b522b.png)
![2017-03-30 6 21 12](https://cloud.githubusercontent.com/assets/6477701/24497344/1b4013ae-1576-11e7-9599-9f91e4f89317.png)